src/main/java/com/google/sps/data/User.java

```diff
@@ -22,11 +32,64 @@ public boolean equals(User user) {
   }
 
   public void addProductArea(String toAdd) {
-      productArea.add(toAdd);
+      specialties.add(toAdd);
+  }
+
+  public Collection<String> getSpecialties() {
+      return specialties;
+  }
+
+  public void setProfileType(String input) {
```

Comment:
this seems error prone in the even that other strings are used (e.g. "Student", ""). Consider taking the ProfileType as an argument and setting it directly.

Reply:
Done